### DISCUSSION TITLE
Configure  affinity

### DISCUSSION THREAD

**Discussion by PauloJFCabral** [Opster/opensearch-k8s-operator#654] [Ideas]
I think could be a good feature to add affinity to the configurations

**Comment by swoehrl-mw** [comment]
Hi @PauloJFCabral. Could you please elaborate on what specifically you want? Because the operator already supports configuring affinity for nodepools and for dashboards pods.

**Comment by vchirikov** [comment]
I don't see `nodeAffinity` section here:

* https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/charts/opensearch-operator/values.yaml
